libcalico-go/lib/backend/k8s/resources/workloadendpoint.go

```diff
@@ -53,6 +53,27 @@ type WorkloadEndpointClient struct {
 	converter conversion.Converter
 }
 
+type patchModeKey struct{}
+type PatchMode string
+
+const (
+	PatchModeCNI           PatchMode = "patchCNI"
+	PatchModeEgressGateway PatchMode = "patchEgressGateway"
```

Comment:
I don't think we normally include unused constants in the OSS codebase - we should just include the constants that are used in Calico and then add the constants needed in Enterprise.

Reply:
Thanks, good catch.